packages/amplify-graphql-transformer-core/src/transform-host.ts

```diff
@@ -186,6 +227,27 @@ export class DefaultTransformHost implements TransformHostProvider {
     dataSourceName?: string,
     pipelineConfig?: string[],
     scope?: Construct,
+  ): CfnResolver => {
+    return this.addResolver(
+      typeName,
+      fieldName,
+      { requestMappingTemplate, responseMappingTemplate },
+      resolverLogicalId,
+      dataSourceName,
+      pipelineConfig,
+      scope,
+      VTL_RUNTIME,
+    );
+  };
+
+  public addResolver = (
```

Comment:
Should this be public, or would we prefer access to come in via one of the JS/VTL specific methods? (Future PR is fine if you agree there needs to be a change)